DataManagementSystem/Agent/RequestOperations/MoveReplica.py

```diff
@@ -0,0 +1,447 @@
+""" MoveReplica operation handler
+
+    This handler moves replicas from source SEs to target SEs. Replicas are first replicated to target SEs and then removed from the source SEs
+"""
+
+__RCSID__ = "$Id $"
+
+# # imports
+import re, os, string
```

Comment:
The `string` module is not deprecated per se, but several of its functionalities are: I see you are using it here for `string.lower()` while it's not needed, just use `.lower()` on a the string object directly.